1_wqp_pull.yml

```diff
@@ -41,8 +42,7 @@ targets:
   # only filters by characteristicNames, and pulls all at the same time
 
   start_year:
-    command: c(I('1900'), I(seq(1980, 1997, by = 3)),
-      I(seq(1998, 2003, by = 2)), I(seq(2004, 2020)))
+    command: get_start_years(begin = 1900, end = wqp_pull_date)
```

Comment:
This does seem cleaner to have the end year set from the wqp_pull_date, and to have the sequencing documented in a function.